app/services/liquid_assigns_service.rb

```diff
@@ -15,4 +15,27 @@ def assigns
 
   def assigns_pro
   end
+
+  def project_drops
+    {
+      'evidence' => project_records(type: :evidence),
+      'issues' => project_records(type: :issue),
+      'nodes' => project_records(type: :node),
+      'notes' => project_records(type: :note),
+      'project' => ProjectDrop.new(project),
+      'tags' => project_records(type: :tag)
+    }
+  end
+
+  def project_records(type:)
+    records = project.send(type.to_s.pluralize)
+    records = records.user_nodes if type == :node
```

Comment:
Instead of fetching the records dynamically, what about passing them into this method as an argument so that it doesn't have to worry about specific cases like if type == :node. Then it only cares about creating the drops & caching them